src/config/dev.ts

```diff
@@ -0,0 +1,27 @@
+import { ConfigInterface } from './config';
+
+const config: ConfigInterface = {
+  http: {
+    host: 'localhost',
+    port: 8088,
+    workerPool: false,
+  },
+  db: {
+    host: 'localhost',
+    user: 'postgres',
+    password: 'test',
+    port: 5432,
+    forceSSL: false,
+    multiUser: false,
+  },
+  logger: {
+    debug: true,
+    test: false,
+    forceLocal: false,
+  },
+  telemetry: {
+    posthogKey: 'phc_xvAQWfpHug7G0SuU5P9wwAbvP9ZawgAfIEZ9FUsiarS',
+  },
```

Comment:
do we want telemetry for dev?

Reply:
no, good point

Reply:
I had just copy-pasted this from the `local` environment, because that's what we were using for the `dev` builds before. But I can remove it here.

Reply:
```suggestion
```